dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.cpp

```diff
@@ -640,7 +635,7 @@ RtpsUdpDataLink::associated(const RepoId& local_id, const RepoId& remote_id,
       RtpsWriter_rch writer = rw->second;
       g.release();
       writer->update_max_sn(remote_id, max_sn);
-      writer->add_reader(make_rch<ReaderInfo>(remote_id, remote_durable));
+      writer->add_reader(make_rch<ReaderInfo>(remote_id, remote_durable, remote_context));
```

Comment:
Can we just rename this variable to participant_flags since that makes it clear what we're dealing with?